.github/workflows/buildCheck.yml

```diff
@@ -0,0 +1,48 @@
+# This workflow will build and package the nodejs service and deploy
+
+name: Foundry UI Build Check
+
+on:
+  pull_request:
+    branches: [master, v1-release, next]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+          registry-url: "https://registry.npmjs.org"
+      - name: Prepare
+        run: npm install
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Lint React
+        run: npm run lint-react
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Are these tokens needed in the steps where we're not installing anything?

Reply:
Right now they are not, but I put them there since they don't hurt anything. The advantage of having them there now is less work to do for CICD if we end up depending on our own private package.